pkgs/development/python-modules/sparklines/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, future
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "sparklines";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "deeplook";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1hfxp5c4wbyddy7fgmnda819w3dia3i6gqb2323dr2z016p84r7l";
+  };
+
+  buildInputs = [ future ];
```

Comment:
future is required at runtime (https://github.com/deeplook/sparklines/blob/v0.4.2/sparklines/__main__.py#L8) and must go to propagatedBuildInputs.

Reply:
Thx for the input. I've also added future to `pysmart-smartx` because of <https://github.com/smartxworks/pySMART/blob/v0.3.10/pySMART/device.py#L25>